scio-core/src/main/scala/com/spotify/scio/values/PairSkewedSCollectionFunctions.scala

```diff
@@ -60,13 +60,12 @@ object SkewedJoins {
 
   // some sensible defaults for skewed joins
   val DefaultHotKeyThreshold: Int = 9000
-  val DefaultHotKeyMethod: HotKeyMethod.Threshold = HotKeyMethod.Threshold(DefaultHotKeyThreshold)
   val DefaultHotKeyFanout: Int = 1
   val DefaultCmsEpsilon: Double = 0.001
   val DefaultCmsDelta: Double = 1e-10
   val DefaultCmsSeed: Int = 42
   val DefaultSampleFraction: Double = 1.0
-  val DefaultSampleWithReplacement: Boolean = true
+  val DefaultSampleWithReplacement: Boolean = false
```

Comment:
@RustedBones  why do we use Poisson as default sampling distribution? It introduces repeats of the same items

Reply:
I just kept previous default to prevent change of behavior.

Reply:
Let's change it in 0.13?